translib/db/db_onchangecache.go

```diff
@@ -0,0 +1,123 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2021 Broadcom. The term Broadcom refers to Broadcom Inc. and/or //
+//  its subsidiaries.                                                         //
+//                                                                            //
+//  Licensed under the Apache License, Version 2.0 (the "License");           //
+//  you may not use this file except in compliance with the License.          //
+//  You may obtain a copy of the License at                                   //
+//                                                                            //
+//     http://www.apache.org/licenses/LICENSE-2.0                             //
+//                                                                            //
+//  Unless required by applicable law or agreed to in writing, software       //
+//  distributed under the License is distributed on an "AS IS" BASIS,         //
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  //
+//  See the License for the specific language governing permissions and       //
+//  limitations under the License.                                            //
+//                                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+package db
+
+import (
+	"github.com/golang/glog"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+//  Internal Types                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+type dbOnChangeReg struct {
+	CacheTables map[string]bool // Only cache these tables.
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//  Exported Functions                                                        //
+////////////////////////////////////////////////////////////////////////////////
+
+func (d *DB) RegisterTableForOnChangeCaching(ts *TableSpec) error {
+	if glog.V(1) {
```

Comment:
Is this message _really_ that important to have level set to 1? As I read the code it is a debugging message, so, level 3 would look good enough.